supertokens_python/supertokens.py

```diff
@@ -133,10 +134,12 @@ def defaultImpl(o: Any):
         return json.dumps(self, default=defaultImpl, sort_keys=True, indent=4)
 
 
-def manage_session_post_response(session: SessionContainer, response: BaseResponse):
+async def manage_session_post_response(
+    session: SessionContainer, response: BaseResponse
+):
     # Something similar happens in handle_error of session/recipe.py
     for mutator in session.response_mutators:
-        mutator(response=response)  # type: ignore
+        await resolve(mutator(response=response))  # type: ignore
```

Comment:
Now that all mutators must be async, you don't need resolve, right?

Reply:
There's one more occurence of this. Fix that as well.

Reply:
@iresharma ^

Reply:
Done